spec/services/dwp/benefit_check_service_spec.rb

```diff
@@ -1,35 +1,131 @@
 require 'rails_helper'
+require 'savon/mock/spec_helper'
 
 RSpec.describe DWP::BenefitCheckService do
-  subject { described_class.call(applicant) }
+  subject { described_class }
 
-  let(:last_name) { 'Smith' }
-  let(:date_of_birth) { '1999/01/11'.to_date }
-  let(:nino) { 'NC123459A' }
+  let(:last_name) { 'Walker' }
+  let(:date_of_birth) { '1980/01/10'.to_date }
+  let(:nino) { 'JA293483A' }
+  let(:id) { '123' }
 
   let(:applicant) do
     double(
       Applicant,
+      id:,
       last_name:,
       date_of_birth:,
-      nino:
+      nino:,
     )
   end
 
+  before do
+    Rails.configuration.x.benefit_checker.use_mock = false
+    Rails.configuration.x.benefit_checker.wsdl_url = 'http://benefit-checker/?wsdl'
```

Comment:
We can add this to the `.env.test` file, akin to what we did with the datastore, and that way we ensure it is always mocked for tests:

```
# DWP check endpoint (stubbed in tests)
BC_WSDL_URL=http://benefit-checker/?wsdl
```

I'm not a big fan of changing `Rails.configuration.x` values in the tests. These values will persist for other tests, and sometimes it produces flaky/intermittent tests difficult to diagnose.

As a suggestion, as you are initialising `BenefitCheckService` with the config, you may be able to stub that config in tests, and avoid having to change the configuration globally.